railties/lib/rails/generators/rails/plugin/templates/app/controllers/%namespaced_name%/application_controller.rb.tt

```diff
@@ -1,6 +1,5 @@
 <%= wrap_in_modules <<~rb
   class ApplicationController < ActionController::#{api? ? "API" : "Base"}
-    #{ api? ? '# ' : '' }protect_from_forgery with: :exception
```

Comment:
Setting `default_protect_from_forgery` to true is in `load_defaults`. 
However, engine does not use `load_defaults`.  
Therefore, I think that this is still necessary.

Reply:
Thanks for the pointing this.